spec/fixtures/activities/ams/newt/newt_real_and_complete_legacy_file.xml

```diff
@@ -0,0 +1,9687 @@
+
```

Comment:
I'm slightly regretting my past decision to use fixtures for the real import. It technically works because we trust that the IATI files aren't changing but it feels misleading to the reader. Do you think it's an issue and if so worth doign something about? Should we have these files timestamped in vendor maybe?

Reply:
Yeah fixtures doesn't feel right for the "real" data tbh. I think moving them to vendor with the IATI codelists is a good shout. 

Should that be done here or in a separate PR moving the UKSA ones as well?

Reply:
Okay let's do it then! The UKSA ship has sailed, the data migration documents what it used at the time so it might be best to leave that as an imperfection and focus on making it better for AMS in this PR.